spec/views/civil_servants/services/show.html.slim_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'civil_servants/services/show.html.slim', type: :view do
+  let(:civil_servant) { create(:civil_servant, :full) }
+  let(:user) { civil_servant.user }
+  let(:service) { create(:service, :future, civil_servant: civil_servant) }
+  let(:service_spec) { service.service_specification }
+  let(:organization) { service.organization }
+  let(:org_address) { organization.address }
+  let!(:expense_sheet) do
+    create(:expense_sheet, :locked,
+           service: service, beginning: service.beginning, ending: service.beginning.at_end_of_month)
+  end
+
+  let(:expected_strings) do
+    [
+      t('civil_servants.services.show.organization_information'),
+      organization.name
+    ] + [
+      org_address.primary_line,
+      org_address.secondary_line,
+      org_address.supplement,
+      org_address.street,
+      org_address.zip_with_city
```

Comment:
Isn't this just `org_address.full_compose`?

Reply:
No because full compose is one string. But on the view the \n are replaced with <br>